composer.json

```
@@ -86,6 +86,7 @@
         "symfony/polyfill-php80": "^1.17",
         "symfony/process": "^4.4",
         "symfony/property-access": "^4.4",
+        "symfony/property-info": "4.4.*",
```

```suggestion
        "symfony/property-info": "^4.4",
```
It won't change much, but I would prefer to keep it consistent with others constraints.